ui/src/i18n/fr.js

```diff
@@ -0,0 +1,112 @@
+import deepmerge from 'deepmerge'
+import frenchMessages from 'fr-language-french'
```

Comment:
I forgot to document you need to install the basic translation for a given language: https://marmelab.com/react-admin/Translation.html#available-locales

In your case, you need to install it with:
```
npm install --save ra-language-french
```

And import it with `import frenchMessages from 'ra-language-french'`